spec/index.bs

```diff
@@ -840,6 +881,66 @@ enum GPUBindingType {
 };
 </script>
 
+  * {{GPUBindGroupLayoutBinding/textureDimension}}, {{GPUBindGroupLayoutBinding/multisampled}}:
+    The dimensions and multi-sampling properties of texture bindings.
+
+    Note: This allows Metal-based implementations to back the respective bind
+    groups with `MTLArgumentBuffer` objects that are more efficient to bind at
+    run-time.
+
+  * {{GPUBindGroupLayoutBinding/hasDynamicOffsets}}:
+    For {{GPUBindingType/uniform-buffer}}, {{GPUBindingType/storage-buffer}}, and {{GPUBindingType/readonly-storage-buffer}} bindings, 
+    indicates that the binding has dynamic offsets. One offset must be passed to 
```

Comment:
as noted in my previous comment, one binding can't have multiple dynamic offsets (today)

Reply:
Addressed!